srcpkgs/cjs/template

```diff
@@ -1,26 +1,21 @@
 # Template file for 'cjs'
 pkgname=cjs
-version=5.4.1
+version=5.8.0
 revision=1
 build_style=meson
-build_helper=gir
+build_helper="gir"
 # XXX: disable tests because they need dbus, X and GTK, same as gjs
 configure_args="-Dskip_dbus_tests=true -Dskip_gtk_tests=true
  -Dprofiler=disabled -Dinstalled_tests=false"
-hostmakedepends="pkg-config gobject-introspection glib-devel dbus mozjs78-devel"
-makedepends="dbus-glib-devel readline-devel gobject-introspection
- gtk+3-devel mozjs78 mozjs78-devel"
+hostmakedepends="pkg-config glib-devel dbus"
+makedepends="dbus-glib-devel readline-devel gtk+3-devel mozjs102
```

Comment:
We may need `gtk4-devel`

Reply:
Reading through meson.build, it seems that gtk4-devel is only needed for tests (and gtk+3-devel doesn't seem to be needed at all).